oap-server/analyzer/meter-analyzer/pom.xml

```diff
@@ -42,6 +42,16 @@
             <groupId>io.vavr</groupId>
             <artifactId>vavr</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.kubernetes</groupId>
+            <artifactId>client-java-api</artifactId>
+            <version>10.0.0</version>
```

Comment:
Please use the version managed by the OAP root pom.xml

Reply:
done